web-server/src/components/WorkflowSelector.tsx

```diff
@@ -126,14 +126,14 @@ export const DeploymentWorkflowSelector: FC<{ repo: BaseRepo }> = ({
         sx={{ textAlign: 'start' }}
       >
         {loading.value ? (
-          <FlexBox alignCenter gap2>
+          <FlexBox alignCenter gap2 sx={{ p: 2 }}>
```

Comment:
can you show relevant snapshots/recordings for this change?

Reply:
Refer this if it helps!
https://github.com/middlewarehq/middleware/blob/765c3d8654138b18e70b57c036f8da91bff8230a/web-server/src/components/TeamsList.tsx#L327-L333

Reply:
Thanks for the reference , I have used the above method and this is how it looks now.

![Screenshot (103)](https://github.com/middlewarehq/middleware/assets/89914602/14c8db44-78e1-4ccf-9577-618a5af36715)